crates/orchestrator/src/spawner.rs

```diff
@@ -126,11 +126,11 @@ where
 
     println!("\n");
     println!("🚀 {}, spawning.... with command:", node.name);
-    println!("{cmd} {}", args.join(" "));
+    println!("{program} {}", args.join(" "));
 
     let spawn_ops = SpawnNodeOptions {
         name: node.name.clone(),
-        command: cmd,
+        program,
```

Comment:
I know that naming is hard, but what are the reason behind calling this `program`?

Reply:
I used what is used in [std::process::Command::new](https://doc.rust-lang.org/std/process/struct.Command.html#method.new) and [tokio::process::Command::new](https://docs.rs/tokio/latest/tokio/process/struct.Command.html#method.new), I think ```program``` is overall a better name than ```command``` because it could either be an executable path (like a script or full binary path) or an executable (present in the PATH or similar), a command fits more for shell built-in commands imho